### ISSUE TITLE
Install of FFmpeg never completes in 3.1.2 upgrade on jessie

### ISSUE THREAD

**Issue by EdNett** [omega8cc/boa#1050]
When attempting the up-stable from 3.1.1 to 3.1.2, the upgrade fails at the FFmpeg install. Here are the only errors we have seen up to the point of the attempted FFmpeg install (which has lasted 5 hours now, with no noticible stress on cpu or ram (99 - 100 % idle and using less than 1/2 system ram):

sysctl: permission denied on key 'net.ipv4.tcp_challenge_ack_limit'

BOA [10:53:17] ==> WARN: This system is not supported: vmware openvz

BOA [11:13:15] ==> INFO: Installed Nginx version nginx/1.11.1, upgrade required
/opt/tmp/boa/lib/functions/nginx.sh.inc: line 198: chattr: command not found

you want to upgrade Aegir Master Instance? [Y/n] y
BOA [12:39:40] ==> INFO: Running Aegir Master Instance upgrade
/opt/tmp/boa/lib/functions/master.sh.inc: line 329: chattr: command not found

/opt/tmp/boa/lib/functions/master.sh.inc: line 374: chattr: command not found

boa info more is here:
https://gist.github.com/EdNett/2106e2d12ed13adbf0f1ab62b5c3eb37

A process list output follows: (what would you recommend us doing to not break boa - cancel the screen session and redo the up-stable?):

server1:~# ps -a
  PID TTY          TIME CMD
 1156 pts/3    00:00:00 mysqld_safe
 1440 pts/3    00:01:17 mysqld
 1441 pts/3    00:00:00 logger
 1868 pts/2    02:18:55 top
19986 pts/3    00:00:02 aptitude
20637 pts/4    00:00:00 ps
26614 pts/0    00:00:01 screen
28478 pts/3    00:00:00 barracuda
30883 pts/3    00:00:05 bash
server1:~# ps
  PID TTY          TIME CMD
20638 pts/4    00:00:00 ps
29553 pts/4    00:00:00 bash
server1:~#

**Comment by EdNett** [comment]
We rebooted the server (vps), removed "FMG" from the extras list, and re-ran barracuda up-stable.

We immediately hit the following errors, which help to explain this issue?:

INFO: Running system packages update...
BOA [18:41:14] ==> WARN: Installed keyring is broken, forced reinstall required

BOA [18:43:51] ==> INFO: Upgrading required libraries and tools...
BOA [18:43:59] ==> WARN: Installed keyring is broken, forced reinstall required

**Comment by pricejn2** [comment]
I'm guessing its hanging here -- https://github.com/omega8cc/boa/blob/master/lib/functions/xtra.sh.inc#L525

```
st_runner "${_INSTAPP} deb-multimedia-keyring" 2> /dev/null
```

I'd try manually installing deb-multimedia-keyring then confirm you can update repos without error.

**Comment by EdNett** [comment]
Thank you for your advice. I appreciate it!

In this case, on this vps, deb-multimedia-keyring cannot be found, so the needed source must never have been properly added, I guess.  I am troubleshooting this currently

**Comment by omega8cc** [comment]
Thanks for the report. This has been fixed in #1052
